### PR TITLE
Willywil548/cappms/package updates

### PR DESCRIPTION
**Pull request by willywil548**

### CODE REVIEW

File: CaPPMSTests/CaPPMSTests.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>.NETCoreApp,Version=v5.0</TargetFramework>
+    <TargetFramework>.net8.0</TargetFramework>
```
Comment (willywil548) [inline CaPPMSTests/CaPPMSTests.csproj]: Is a breaking change and needs to be resolved.

**Review by mikecollins08 — APPROVED** [review]

File: CaPPMS/Startup.cs [hunk 2]
```diff
@@ -25,18 +24,16 @@ public Startup(IConfiguration configuration)
...
 
             services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
                 .AddMicrosoftIdentityWebApp(Configuration)
                 .EnableTokenAcquisitionToCallDownstreamApi(initialScopes)
                 .AddMicrosoftGraph(Configuration.GetSection("Graph"))
                 .AddInMemoryTokenCaches();
 
-            string tenantId = Configuration.GetValue<string>("AzureAd:TenantId");
-            services.Configure<MicrosoftIdentityOptions>(
-               options => { options.ClientSecret = GetClientSecret(); });
```
Comment (willywil548) [inline CaPPMS/Startup.cs:38]: Not called after package update

File: CaPPMS/Startup.cs [hunk 3]
```diff
@@ -86,14 +83,14 @@ private string GetClientSecret()
             // return string.Empty;
 
             // Used for Production.
-            string clientId = System.Environment.GetEnvironmentVariable("GRAPH_SECRET");
+            string secret = System.Environment.GetEnvironmentVariable("GRAPH_SECRET");
```
Comment (willywil548) [inline CaPPMS/Startup.cs:82]: Is secret not client id

File: CaPPMS/appsettings.json [hunk 4]
```diff
@@ -20,7 +20,7 @@ For more info see https://aka.ms/dotnet-template-ms-identity-platform
       Applications are registered in the https:portal.azure.com portal.
     */
     "BaseUrl": "https://graph.microsoft.com/v1.0",
-    "Scopes": "user.read Directory.AccessAsUser.All"
+    "Scopes": "user.read,Directory.AccessAsUser.All"
```
Comment (willywil548) [inline CaPPMS/appsettings.json:23]: comma separated is more intuative

File: CaPPMS/wwwroot/css/site.css [hunk 5]
```diff
@@ -14,12 +14,21 @@ a,
     color: #fff;
     background-color: #da1a32;
     border-color: #a30606;
+    margin:3px
```
Comment (willywil548) [inline CaPPMS/wwwroot/css/site.css:17]: User readability

File: CaPPMSTests/ComponentTests/BunitTestHelper.cs [hunk 6]
```diff
@@ -16,26 +14,8 @@ public void Setup()
...
-
-            // IConfiguration needed once the Consent Form logic can be mocked
-            //IConfigurationRoot configuration = new ConfigurationBuilder()
-            //.AddJsonFile("appsettings-test.json")
-            //.Build();
-
-            //TestContext.Services.AddSingleton<IConfiguration>(configuration);
-            TestContext.Services.AddSingleton(new ProjectManagerService("Component.tests.json"));
-            TestContext.Services.AddOptions();
-            TestContext.Services.AddRazorPages()
```
Comment (willywil548) [inline CaPPMSTests/ComponentTests/BunitTestHelper.cs:29]: package updates required update here as well

**Review by willywil548 — COMMENTED** [review]
LGTM

**Pull request merged**